lms/djangoapps/certificates/admin.py

```diff
@@ -28,6 +29,11 @@ def __init__(self, *args, **kwargs):
         self.fields['organization_id'] = forms.TypedChoiceField(
             choices=org_choices, required=False, coerce=int, empty_value=None
         )
+        lang_choices = all_languages()
+        lang_choices.insert(0, ('', '------------'))
+        self.fields['language'] = forms.ChoiceField(
+            choices=lang_choices, required=False
+        )
```

Comment:
Does Django perform any validation on this field after the form is submitted?

Reply:
Do you mean in terms of whether or not the language is supported?

Reply:
The only languages that will be available for selection will be those listed in the DarkLang configuration, which should only contain supported langauges